test/bit_string_test.exs

```diff
@@ -0,0 +1,77 @@
+defmodule BitStringTest do
+  use ExUnit.Case, async: true
+  import Postgrex.TestHelper
+  alias Postgrex, as: P
+
+  setup do
+    opts = [database: "postgrex_test", backoff_type: :stop,
+            extensions: [{Postgrex.Extensions.BitString, []}]]
```

Comment:
We want to avoid requiring the extension explicitly. Please add the extension module to the attribute in `Postgrex.Utils`.